cardano_node_tests/tests/tests_conway/test_pparam_update.py

```diff
@@ -1175,6 +1186,122 @@ def _check_state(state: dict):
         if fin_voted_votes.drep:
             governance_utils.check_vote_view(cluster_obj=cluster, vote_data=fin_voted_votes.drep[0])
 
+        # Check for deposit return
+        deposit_amt = cluster.conway_genesis["govActionDeposit"]
+        enact_deposit_returned = cluster.g_query.get_stake_addr_info(
+            pool_user_lg.stake.address
+        ).reward_account_balance
+        assert (
+            enact_deposit_returned
+            == init_return_account_balance + deposit_amt * submitted_proposal_count
+        ), "Incorrect return account balance"
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    @pytest.mark.smoke
```

Comment:
We mark tests as "smoke" if they can finish under 2 mins. This one needs to wait for epoch boundary, so it is not a smoke test by our definition.

Reply:
Thanks for the information. Latest update has been pushed to integrated into the test_pparam_update itself.